static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { mocked } from 'ts-jest/utils';
+import raven_ from 'lib/raven';
+import config_ from '../../lib/config';
+import { amIUsed } from './sentinel';
+
+const raven = raven_;
+
+const config = config_ as {
+	get: (s: string, d: boolean) => boolean;
+};
```

Comment:
Ditto for importing mocks, you can ignore them entirely, I think.

```suggestion

import { amIUsed } from './sentinel';

```